glib/adbc-glib/meson.build

```diff
@@ -82,7 +82,7 @@ pkgconfig.generate(libadbc_glib,
                    description: 'GLib API for ADBC driver manager',
                    filebase: 'adbc-glib',
                    name: 'ADBC GLib',
-                   requires: ['gobject-2.0', 'adbc-driver-manager'],
+                   requires: ['gobject-2.0', 'adbc_driver_manager'],
```

Comment:
Why do we need this change?
We're using `adbc-driver-manager.pc`.

Reply:
When not using pkgconfig and letting the wrap entry download this for you, the name of the lib is just adbc_driver_manager, so this was my best attempt at working around the inconsistency between the pkgconfig name and library name. 

Could be a misunderstanding - need to investigate more